plugin/mutate/data/dashboard.html

```diff
@@ -12,6 +12,16 @@
   </head>
 
   <body>
+    <div id ="csp_error">
+      <h2>Something isn't right...</h2>
```

Comment:
Maybe add what the problem is.
"Something isn't right... Javascript need to be enabled"